templates/layout-grid-slider.php

```diff
@@ -2,7 +2,12 @@
 namespace Netzstrategen\Gallerya;
 
 $group_size = 6;
-$image_attr = Plugin::isLazyLoadActive() ? ['data-no-lazy' => '1'] : [];
+$slider_image_size = has_image_size('post-thumbnail') ? 'post-thumbnail' : 'large';
+// Prevent wrong images height calculation caused by lazy loading.
+$image_attr = apply_filters('gallerya_lazyload_image_attributes', [
+  'data-no-lazy' => '1',
+  'class' => 'no-lazy attachment-large size-' . $slider_image_size,
```

Comment:
The layout-slider template uses `$slider_image_size` also for `attachment-`, this layout-grid-slider template here does not.  I assume one of both is correct and the other isn't, but I don't understand the purpose / effect of the `attachment-` class (which is a documentation problem that could/should be fixed in code, too).